src/diffusers/models/unet_2d_condition.py

```diff
@@ -111,6 +111,9 @@ def __init__(
         dual_cross_attention: bool = False,
         use_linear_projection: bool = False,
         num_class_embeds: Optional[int] = None,
+        resnet_time_scale_shift: str = "default",
+        downsample_resnet: bool = False,
```

Comment:
would maybe rename this to `downsample_type=["default", "resnet"]` . Also wondering if we could just have one parameter instead of two *i.e.* up_down_sample_type -> or will there ever be a case where up and down sample type would differ?

Reply:
Will add both!